PostGreKeyRotation/PostGreKeyRotation/Models/PostgreSQLControllerConfig.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PostGreKeyRotation.Models
+{
+    public class PostgreSQLControllerConfig
```

Comment:
Not to nitpick but perhaps this file should be my commit?

Reply:
Fixed